02-peer-protocol.md

```diff
@@ -1106,6 +1138,10 @@ it's simplest to only allow it to set fee levels; however, as the same
 fee rate applies to HTLC transactions, the receiving node must also
 care about the reasonableness of the fee.
 
+With `option_anchor_outputs`, `feerate_per_kw` is not as critical anymore to
```

Comment:
Possibly move this to be positioned _above_ the guidance for 5x fees? The whole anchor outputs thing invalidates the first 3 paragraphs in this section. We may want to just go with a complete re-write now, and refer to the existing fee considerations as only applying to the legacy commitment format.

Reply:
Yes, better. Moved